addons/account/models/account_move.py

```diff
@@ -912,7 +912,7 @@ def _compute_name(self):
     @api.depends('date', 'journal_id', 'move_type', 'name', 'posted_before', 'sequence_number', 'sequence_prefix', 'state')
     def _compute_name_placeholder(self):
         for move in self:
-            if (not move.name or move.name == '/') and not move._get_last_sequence():
+            if (not move.name or move.name == '/') and not move._get_last_sequence() and self.date:
```

Comment:
```suggestion
            if (not move.name or move.name == '/') and self.date and not move._get_last_sequence():
```
Evaluate the date before `_get_last_sequence`:
* it's cheaper
* `_get_last_sequence` could also require the date